src/modules/devops-guide/pages/system-requirements.adoc

```diff
@@ -124,11 +124,19 @@ If you enable OAuth2 or OIDC sign-up or access external services via automation
 
 == Domain and Hostname
 
-.The most common setup involves two domains -- API and web app:
-* Hostname for Web application,
-* hostname for API server.
+The most common way to set up your {PRODUCT_NAME}  instance is by hosting the web application and the API server on one domain and providing;
```

Comment:
for the most common and effortless setup we recom. using Corteza docker image that contain web applications and server (all-in-one) and both served from the same domain.